scalableminds/webknossos
# Check if mesh is exists before saving

This PR adds a check before trying to save an isosurface as an stl file. This ensures that there is no error when not having loaded any isosurfaces. 
A better solution would be to simply disable the download button if no isosurface is loaded. But this is not so easy to implement as the isosurfaces are saved in the `scene_controller` and informing the meshes-tab about any changes of the isosurfaces is not done in a short amount of time. Thus I would skip this for now.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a hybrid or volume tracing and do not load any isosurface.
- Click the download button in the meshes tab -> nothing should happen (nor error popping up).

### Issues:
- fixes #4566

------
